Documentation/ApiOverview/Events/Events/Backend/AfterRecordSummaryForLocalizationEvent.rst

```diff
@@ -0,0 +1,61 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; AfterRecordSummaryForLocalizationEvent
+..  _AfterRecordSummaryForLocalizationEvent:
+
+======================================
+AfterRecordSummaryForLocalizationEvent
+======================================
+
+..  versionadded:: 12.0
+
+The event is fired in the
+:php:`\TYPO3\CMS\Backend\Controller\Page\RecordSummaryForLocalization` class
+and allows extensions to modify the payload of the :php:`JsonResponse`.
+
+
+Example
+=======
+
+Registration of the event in your extension's :file:`Services.yaml`:
+
+..  code-block:: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+    Vendor\MyExtension\EventListener\Backend\MyEventListener:
+        tags:
+            - name: event.listener
+              identifier: 'my-extension/backend/after-record-summary-for-localization'
+
+The corresponding event listener class:
+
+..  code-block:: php
+    :caption: EXT:my_extension/Classes/Backend/MyEventListener.php
+
+    use TYPO3\CMS\Backend\Controller\Event\AfterRecordSummaryForLocalizationEvent;
+
+    final class MyEventListener
+    {
+        public function __invoke(AfterRecordSummaryForLocalizationEvent $event): void
+        {
+            // Get current records
+            $records = $event->getRecords();
+
+            // remove or add $records available for translation
+
+            // set new records
+            $event->setRecords($records);
+
+            // Get current columns
+            $columns = $event->getColumns();
+
+            // remove or add $columns available for translation
+
+            // set new columns
```

Comment:
I would remove the comments as they do no more then repeat the method names

Reply:
I adjusted them and would leave them that way for clarity.